examples/integration/demo/README.md

```diff
@@ -0,0 +1,45 @@
+# Astro Starter Kit: Minimal
```

Comment:
This appears to be the same README file as the minimal starter kit. I'm assuming this should be customized for this starter?

Reply:
Yep!  I ended up removing the demo project and added a main README for the integration example (based on the component starter)